File: Travel.Data/Message.cs
using Travel.Common.Enums;

namespace Travel.Data
{
	public class Message
	{
		public int Id { get; set; }
		public int UserId { get; set; }
		public string UserEmail { get; set; }
		public int MessengerId { get; set; }
		public string MessengerDisplayName { get; set; }
		public string MessengerAvatar { get; set; }
		public string MessengerEmail { get; set; }
		public int TravelId { get; set; }
		public string Theme { get; set; }
		public string MessageText { get; set; }
		public bool State { get; set; }
		public DirectionType Direction { get; set; }
		public LuggageType Luggage { get; set; }
		public int PersonCount { get; set; }
		public MessageType Type { get; set; }
	}
}

 

File: Travel.Web/Scripts/Application/Models/Message.ts

/// <reference path="../_references.ts" />
module Application {
    export class Message {
        Id: number;
        UserId: number;
        UserEmail: string;
        MessengerId: number;
        MessengerDisplayName: string;
        MessengerAvatar: string;
        MessengerEmail: string;
        TravelId: number;
        Theme: string;
        MessageText: string;
        State: boolean;
        Direction: DirectionType;
        Luggage: LuggageType;
        PersonCount: number;
        Type: MessageType;
    }
    export enum LuggageType {
        Default = 0, Min = 1, Norm = 2, Max = 3
    }
    export enum DirectionType {
        One = 1 ,Two = 2
    }
    export enum MessageType {
        Subscribe = 1, Accept = 2, Decline = 3, Inner = 4, Outer = 5
    }
}










